internal/pkg/service/buffer/sink/tablesink/storage/config_test.go

```diff
@@ -33,10 +35,9 @@ func TestConfig_With(t *testing.T) {
 			Concurrency:    10,
 		},
 	}
-	volumesAssigmentCfg := local.VolumesConfig{
-		Count:                  2,
-		PreferredTypes:         []string{"foo", "bar"},
-		RegistrationTTLSeconds: 10,
```

Comment:
`RegistrationTTLSeconds` has been moved and cannot be patched - modified from the Sink configuration/